turbo.json

```diff
@@ -13,26 +10,62 @@
       ],
       "outputLogs": "new-only",
       "outputs": [
-        "src/artifacts/**",
-        "src/gen/**",
-        "gen/**"
+        "src/artifacts/**"
+       ]
+    },
+    "compact:fast": {
+      "env": [
+        "COMPACT_HOME",
+        "SKIP_ZK"
+      ],
+      "inputs": [
+        "src/**/*.compact"
+      ],
+      "outputLogs": "new-only",
+      "outputs": [
+        "src/artifacts/**"
       ]
     },
     "test": {
-      "outputs": [],
+      "dependsOn": [
+        "compact:fast",
+        "^compact:fast"
+      ],
+      "env": [
+        "COMPACT_HOME"
+      ],
       "cache": false
     },
     "build": {
       "dependsOn": [
-        "^build",
-        "compact"
+        "^build"
       ],
       "env": [
         "COMPACT_HOME"
       ],
       "inputs": [
         "src/**/*.ts",
-        "src/**/*.mts",
+        "src/**/*.tsx",
+        "!src/**/*.test.ts",
+        "!tests/**/*.ts",
+        "tsconfig.json",
+        "tsconfig.build.json",
+        ".env"
+      ],
+      "outputs": [
+        "dist/**"
+      ]
+    },
+    "build:fast": {
+      "dependsOn": [
+        "^build:fast"
+      ],
```

Comment:
Kinda nitpickish: We're defining this task here but not as a script in the root manifest. This means that this can only be run when the pwd is `contracts/` (or using turbo directly). IMO this makes the command less discoverable and a little confusing. I'd expect `build:fast` to be in root and just not defined in the app package if it's not applicable...though, it could just run the normal `next build` since it's quite quick. Thoughts?